checkita-core/src/main/scala/ru/raiffeisen/checkita/storage/Tables.scala

```diff
@@ -52,6 +53,17 @@ class Tables(val profile: JdbcProfile) {
     def getUniqueCond(r: R): Rep[Boolean] =
       jobId === r.jobId && checkId === r.checkId && referenceDate === r.referenceDate
   }
+
+  sealed abstract class JobResultTable[R <: ResultJobConfig](tag: Tag, schema: Option[String], table: String)
```

Comment:
Looks like this abstract class is overabundant: there is only one class that extends this one and we can extend it directly from DQTable.